### PR TITLE
Update css for improving design

### PR DESCRIPTION
**Pull request by sirius651**
- Fixed : #683 
https://github.com/bancorprotocol/webapp/pull/677#issuecomment-742390106
https://github.com/bancorprotocol/webapp/pull/677#issuecomment-742454448

### CODE REVIEW

File: src/components/vote/proposals/LayoutProposals.vue [hunk 1]
```diff
@@ -59,8 +59,10 @@
             :value="item[column.key]"
           >
             {{ item[column.key] }}
-          </slot>
+          </slot>
```
Comment (H34D) [inline src/components/vote/proposals/LayoutProposals.vue:62]: you can avoid committing this when you run `yarn lint` it will also auto format

Comment (sirius651) [reply src/components/vote/proposals/LayoutProposals.vue:62]: yeah I think this linting task can be executed automatically when pushing.

Comment (H34D) [reply src/components/vote/proposals/LayoutProposals.vue:62]: yes, we can do that but linting takes quite long

Comment (pingustar) [reply src/components/vote/proposals/LayoutProposals.vue:62]: I recommend setting up your IDE to lint/prettier on save.

Comment (sirius651) [reply src/components/vote/proposals/LayoutProposals.vue]: as @H34D said, linting task took heavy long. I handled it manually.

**Review by pingustar — APPROVED** [review]
LGTM.

this is just linting @sirius651 ? Going forward, feel free to merge smaller linting like this directly - no review required for such a small change.

**Pull request merged**